### PR TITLE
update docs and readme

### PR DESCRIPTION
**Pull request by CunliangGeng**
**Major changes**

Update docs
- update info about pre-release and its installation
- add disk space requirement for installation
- add doc about logging

Update README.md
-  update info about installation
- add guideline for testing

### CODE REVIEW

**Comment by CunliangGeng** [comment]
* **#244** <a href="https://app.graphite.dev/github/pr/NPLinker/nplinker/244?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CunliangGeng and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/index.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ For a deep understanding of NPLinker, please refer to the [original paper](https
 
 !!! warning "Under Development"
     
-    NPLinker v2 is under active development. The documentation is not complete yet. If you have any 
-    questions, please contact us via [GitHub Issues](https://github.com/NPLinker/nplinker/issues)
\ No newline at end of file
+    NPLinker v2 is under active development ([See its pre-releases](https://pypi.org/project/nplinker/#history)).
```
Comment (gcroci2) [inline docs/index.md]:
```suggestion
    NPLinker v2 is under active development ([see its pre-releases](https://pypi.org/project/nplinker/#history)). 
```

File: docs/install.md [hunk 2]
```diff
@@ -17,14 +19,15 @@ python --version
...
+# install nplinker package (requiring ~300MB of disk space)
+pip install nplinker==2.0.0a1 # (2)! 
 
-# install nplinker non-pypi dependencies and databases
+# install nplinker non-pypi dependencies and databases (~4GB)
 install-nplinker-deps
 ```
 
 1. A virtual environment is ***required*** to install the the non-pypi dependencies. You can also use `conda` to create a new environment. But NPLinker is not available on conda yet.
+2. NPLinker v2 is still under development and released as [pre-releases](https://pypi.org/project/nplinker/#history). To install the pre-release, you have to explicitly specifiy the version. Using `pip install nplinker` will install the legacy NPLinker (v1.3.2), which is not recommended.
```
Comment (gcroci2) [inline docs/install.md]:
```suggestion
2. NPLinker v2 is still under development and released as [pre-release](https://pypi.org/project/nplinker/#history). To install the pre-release, you have to explicitly specifiy the version. The command `pip install nplinker` will install the legacy NPLinker (v1.3.2), which is not recommended. 
```

File: docs/logging.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+your script. By default, NPLinker will not log any messages. However, you can set up logging in your
+script to log messages. 
+
+```python title="Set up logging in 'your_script.py'"
+# Set up logging configuration first
+from nplinker import setup_logging
+
+setup_logging(level="DEBUG", file="nplinker.log", use_console=True) # (1)!
+
+# Your busniess code here
```
Comment (gcroci2) [inline docs/logging.md]:
```suggestion
# Your business code here
```

**Review by gcroci2 — APPROVED** [review]
Looks great! I added very minor suggestions

**Comment by CunliangGeng** [comment]
### Merge activity

* **May 21, 4:28 AM EDT**: @CunliangGeng started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/NPLinker/nplinker/244).
* **May 21, 4:28 AM EDT**: @CunliangGeng merged this pull request with [Graphite](https://app.graphite.dev/github/pr/NPLinker/nplinker/244).

**Pull request merged**